testing/btest/cluster/websocket/zeromq/oneshot.zeek

```diff
@@ -0,0 +1,71 @@
+# @TEST-DOC: Connect with a WebSocket client, it sends 100 events and disconnects immediately. Ensure the manager sees all of them. Regression test for ZeroMQ discarding queued messages.
```

Comment:
It's tempting to consider merging these two as they're so similar. But, totally a detail.

Reply:
Yeah. It'd be nice to parameterize these as some of the tests should just work for Broker or ZeroMQ - the difference is mostly setup/port usage.

I haven't yet tried as I don't obviously have a good pattern in mind.

To a degree, this applies to storage, too. Not for this PR, but I'd be more than happy if we could figure out something here.